### DISCUSSION TITLE
微信小程序 一个后台服务关联多个后台服务

### DISCUSSION THREAD

**Discussion by John0King** [JeffreySu/WeiXinMPSDK#2615] [Q&A] [answered]
Sample 里面写着可以注册多个

https://github.com/JeffreySu/WeiXinMPSDK/blob/5936d36f975f28c652fdbbba8321392aa4e71a04/Samples/WxOpen/Senparc.Weixin.Sample.WxOpen/Program.cs#L23-L31

但是https://github.com/JeffreySu/WeiXinMPSDK/blob/5936d36f975f28c652fdbbba8321392aa4e71a04/Samples/WxOpen/Senparc.Weixin.Sample.WxOpen/Controllers/WxOpenController.cs#L39-L42 
里面好像只能用一个

**Comment by JeffreySu** [comment, marked as answer]
您可以访问 Config.SenparcWeixinSetting.Items，设置任意多组账号。也可以使用 Senparc.Weixin.All 中的方式，自动注册所有账号：

https://github.com/JeffreySu/WeiXinMPSDK/blob/75de2909c54620cb2b5fa900ebf50f7f6d31c6d4/Samples/MP/Senparc.Weixin.Sample.MP.Simple/Program.cs#L23C1-L30C11
